sweetpotato_trait.obo

```diff
@@ -3,6 +3,8 @@ date: 03:06:2026 10:50
 saved-by: Naama
 namespace-id-rule: * CO_331:$sequence(7,0000000,0009999)$
 ontology: CO_331
+default-namespace: SweetpotatoTrait
```

Comment:
need to change the default namespace to something else (e.g. 'sweetpotato') that is not the namespace we use for traits and variables. This is because Protege software will mess up the namespace field if it is identical to the default one in the header